simuleval/evaluator/remote.py

```diff
@@ -4,8 +4,23 @@
 # This source code is licensed under the license found in the
 # LICENSE file in the root directory of this source tree.
 
+import sys
 import logging
-from simuleval.data.segments import Segment, segment_from_json_string
+import threading
+import time
+from queue import Queue
+
```

Comment:
The import here will run at the beginning so we need to add the dependency to run them. Could you try something like

```python
try:
    import wave
    import pyaudio
    from silero_vad import load_silero_vad, read_audio, 
    get_speech_timestamps
except:
    wave, pyaudio, silero_vad = None, None, None
```

And later in the __init__function to check is the package is availbable

Reply:
Can't we just assume that the user has the correct dependencies to import these packages? If not, they should get an error and they'd know to pip install them. We could also put wave, pyaudio, and silero_vad in setup.py. What do you think?

Reply:
Hi @Epic-Eric the reason is that some people might just use the minimal function for evaluation and we try to make SimulEval light weight. wave, pyaudio, and silero_vad are third party package which only required for CL demo. 

Instead of an error on the root level, it would be better we through this error inside the demo. For example, we can add this into the demo function

```python
if None in [wave, pyaudio, silero_vad]:
    raise Exception('Please install wave, pyaudio, silero_vad to run the demo.')
```